cmd/charm/charmcmd/attach.go

```diff
@@ -11,18 +11,17 @@ import (
 	"github.com/juju/cmd"
 	"gopkg.in/errgo.v1"
 	"gopkg.in/juju/charm.v6-unstable"
-	"gopkg.in/juju/charmrepo.v2-unstable/csclient/params"
 	"launchpad.net/gnuflag"
 )
 
 type attachCommand struct {
 	cmd.CommandBase
 
-	channel string
-	id      *charm.URL
-	name    string
-	file    string
-	auth    authInfo
+	cv   chanValue
```

Comment:
I think I'd still prefer "channel" as a field name here over cv, even though it's not strictly a params.Channel.